package.json

```diff
@@ -48,7 +48,7 @@
     "@babel/plugin-transform-runtime": "^7.13.9",
     "@babel/plugin-transform-typescript": "^7.13.0",
     "@babel/polyfill": "^7.11.5",
-    "@babel/preset-env": "^7.12.0",
+    "@babel/preset-env": "^7.16.4",
     "@babel/runtime": "7.12.18",
```

Comment:
Might be worth it to try and unpin @babel/runtime again after this PR too (Discussion in #384 )